photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -134,21 +135,20 @@ object RandomEffectCoordinate {
    *
    * @note The score is the dot product of the model coefficients with the feature values (in particular, does not go
    *       through non-linear link function in logistic regression!).
-   *
    * @param randomEffectDataSet The active dataset to score
    * @param randomEffectModel The model to score the dataset with
    * @return The computed scores
    */
   protected[algorithm] def score(
     randomEffectDataSet: RandomEffectDataSet,
-    randomEffectModel: RandomEffectModel): KeyValueScore = {
+    randomEffectModel: RandomEffectModel): SimpleDataScores = {
 
     val activeScores = randomEffectDataSet
       .activeData
       .join(randomEffectModel.modelsRDD)
```

Comment:
I'm definitely open to copying the mentioned documentation to additional locations - where do you think is best?